src/index.html

```diff
@@ -15,8 +15,95 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img
+        class="imgblock"
+        src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1730073600&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=GFjFD6-6KNqFBkswmu-BuTPJnotQ8-MD~XFWAHKRTFzBEQqKLe-4Cca9JX55r9scHqqYuwSS3TsIyJCh2iERYUgPUe5fszbMK7Tg0AgZZdstjYggVmO~lCJAm5RABG4y~Pd9KlhMIhkeNGbSjrNGNjL~pLc8xnAb1HFOb19z7kl6MngmQbQxT-1vuKf03PUIUzsEhIR8aicMa2KjYgomPp2TDMN9G7n8WqTgmW1h5Qu-jctRoMRYuvHf4xX9pw5TT2~LSzqKDtR0mjg38R~TXmLshaA9MgAoE7RyK873WOssIDG5MfRQ236zHRF9tZrkvuRgIxDAGL6O9x7hCoKsAg__"
+        alt="blue_logo_40x40"
+      />
+      <nav>
+        <ul class="navItems">
+          <li class="navListItem">
+            <a
+              href="#"
+              class="navLink is-active"
+            >
```

Comment:
The `href` attribute in the `a` tag shouldn't have a space. It should start with the `#` symbol. The `#` symbol is used for anchor links which is a link that allows you to link to a specific part of a page.